packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx

```diff
@@ -0,0 +1,377 @@
+import React from 'react';
+import { Form, Formik, FormikHelpers, FormikValues } from 'formik';
+import { GetSettings, IdentityVerificationAddDocumentResponse, ResidenceList } from '@deriv/api-types';
+import { Button, DesktopWrapper, HintBox, Loading, Text } from '@deriv/components';
+import {
+    filterObjProperties,
+    isEmptyObject,
+    removeEmptyPropertiesFromObject,
+    toMoment,
+    WS,
+    idv_error_statuses,
+    TIDVErrorStatus,
+    IDV_NOT_APPLICABLE_OPTION,
+    isMobile,
```

Comment:
Im not sure but arent we supposed to be using `is_mobile` from ui store ?

Reply:
I'm not sure the reason of implementing is_mobile in ui_store, but we everywhere use isMobile from shared

Reply:
is there any difference or purpose of having them both?

Reply:
as it requires adding additional prop to this component, we will replace it later with refactoring to useStore